UnityXrefMaps/Program.cs

```diff
@@ -156,7 +158,13 @@ private static void FixCsprojForDocFx(string csprojPath)
 
             // Generate site and xref map
             Console.WriteLine($"Running DocFX on '{commit}'");
-            Utils.RunCommand("docfx", output => Console.WriteLine(output), error => Console.WriteLine(error));
+            int docfxExitCode = Utils.RunCommand(
+                "docfx",
+                output => Console.WriteLine("[DocFX] " + output),
+                error => Console.WriteLine("[DocFX - Error] " + error)
+            );
+            if (docfxExitCode != 0)
+                throw new Exception($"docfx command returned exit code {docfxExitCode}");
```

Comment:
I would like to keep the same braces style across all the project 

```suggestion
            {
                throw new Exception($"docfx command returned exit code {docfxExitCode}");
            }
```